assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -158,12 +148,16 @@ function ExecutionResults({
         onFilterChange={(newPredicates) => setPredicates(newPredicates)}
       />
       <ResultsContainer
-        catalogError={false}
+        error={catalogError || executionError}
+        errorContent={[
+          `Failed loading catalog: ${catalogError}`,
```

Comment:
This now has the issue that if we only have one error, we would still have 2 messages, right?
Maybe we could have something like this, with the spread operator:
```
errorContent{[
  ... catalogError ? [`Failed loading catalog: ${catalogError}`] : [],
  ... executionError ? [`Failed loading execution: ${executionError}`]: []
]}
```

Reply:
Not really, the `NotificationBox` filters out falsy entries in that list https://github.com/trento-project/web/pull/1232/files#diff-b5395dece1ad14cf45ccd7203b1761e3a873ea659351067b200654f3e0c7dcd4R8

Would that be enough?

Reply:
my bad, you're right. There will always be something in the way I did :facepalm: Fixing

Reply:
ended up with 
```
errorContent={[
  catalogError ? `Failed loading catalog: ${catalogError}` : null,
  executionError ? `Failed loading execution: ${executionError}` : null,
]}
```